lib/service_base_url_test_kit.rb

```diff
@@ -81,6 +81,10 @@ class ServiceBaseURLTestSuite < Inferno::TestSuite
         label: 'Open Source',
         url: 'https://github.com/inferno-framework/service-base-url-test-kit'
       },
+      {
+        label: 'Download',
+        url: 'https://github.com/inferno-framework/service-base-url-test-kit/releases'
```

Comment:
I was putting trailing slashes on the label urls.  Is that required?

Reply:
It doesn't seem to matter for Inferno or for GitHub. Both work and I don't know of any convention encouraging one over the other in this case.